doc/introduction/circuits.rst

```diff
@@ -70,8 +66,8 @@ constraints:
 
 .. note::
 
-    The quantum operations cannot be used outside of a quantum circuit function, as all
-    :class:`Operations <pennylane.operation.Operation>` must be executed from within a QNode.
+    :doc:`operations` can be created outside of a quantum circuit function; however,
+    quantum operations can only be *executed* from within a QNode.
```

Comment:
Thinking about this, do we want to mention at all that operations can be created outside of a quantum circuit function? The use case for that is quite small, and may lead to odd patterns being attemped.

Reply:
How about just:
```rest
.. note::

    Quantum operations can only be executed on a device from within a QNode.
```

Reply:
Had the same thought.